Tests/function/tests/Add-VSTeamProject.Tests.ps1

```diff
@@ -88,6 +88,7 @@ Describe 'VSTeamProject' {
                Typeid = '00000000-0000-0000-0000-000000000002'
             }
          }
+         [vsteam_lib.ProcessTemplateCache]::Invalidate()
```

Comment:
why do you need to invalidate the cache here? It is just a unit test, I thought that should not be needed.

Reply:
The cache is an object which is not cleared between different unit tests. So if test 1 assumes it is working with a custom process named "Scrum MkII" and only populates the cache with that ,and test assumes it is working with the built in template "Agile" it will may "Agile is not a valid template" if he cache is not cleared.